iotedgehubdev/composeproject.py

```diff
@@ -33,7 +35,7 @@ def compose(self):
         modules.update(self.module_content['$edgeAgent']['properties.desired']['modules'])
         for service_name, config in modules.items():
             self.Services[service_name] = {}
-            create_option_str = config['settings']['createOptions']
+            create_option_str = ComposeProject._join_create_options(config['settings'])
```

Comment:
also for system modules

Reply:
System modules (i.e., edgeHub) is already covered (https://github.com/Azure/iotedgehubdev/pull/99/files#diff-11647d7f78ebc8ad0d6859d0ff403c5bR35)